### DISCUSSION TITLE
能否增加一个快捷键针对单段落执行第二个翻译引擎的翻译

### DISCUSSION THREAD

**Discussion by qianzyGit** [immersive-translate/immersive-translate#1444] [Ideas]
### 新功能 | New Feature

1. 第二翻译引擎设置
2. 绑定快捷键
3. 翻译结果再次增加标签附加到之前的翻译结果之后

### 为什么您想要添加这些功能 | Why you want to add these features

1. 我的使用习惯是默认使用google进行全量翻译,但是翻译质量堪忧
2. 所以我想对定向的段落进行其他翻译引擎的调用
3. 配置openai的api全量翻译会频繁触发429 ,且不能保证显示之前的google的其他的翻译内容

**Comment by keyboard3** [comment]
你可以给段落悬停功能设置单独的翻译服务，在默认的翻译服务翻译全量内容之后，可以用悬停功能2次，还原并再次重新用悬停的服务重新翻译
